readthedocs/projects/urls/private.py

```diff
@@ -84,7 +83,9 @@
     ),
     re_path(
         r"^(?P<project_slug>[-\w]+)/advanced/$",
-        ProjectAdvancedUpdate.as_view(),
+        login_required(
+            RedirectView.as_view(pattern_name="projects_edit", permanent=True),
```

Comment:
```suggestion
            RedirectView.as_view(pattern_name="projects_edit"),
```

Just in case we want to use `/advanced/` URL for a different thing.

Reply:
Thoughts @agjohnson?

Reply:
Good question. I lean slightly towards non-permanent just because it won't be a heavily used redirect anyways. I don't see us reusing the URL either though, so permanent does seem safe.